scripts/import_booze/import_booze.py

```diff
@@ -60,93 +103,27 @@ def _map_description(status):
 Some variants of this item may contain animal products. 
 Check the provided sources for further details.
 '''
-    return None
-
-
-def _map_url(url):
-    return _BARNIVORE_URL + url
+    return ''
```

Comment:
Why do you return an empty string?

Reply:
d1df5ccd3f1fe031ba5fe6475ffa52419304ad6d